modules/consensus/setdb.go

```diff
@@ -0,0 +1,226 @@
+package consensus
+
+import (
+	"errors"
+
+	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/Sia/crypto"
+	"github.com/NebulousLabs/Sia/encoding"
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/persist"
+	"github.com/NebulousLabs/Sia/types"
+	"github.com/boltdb/bolt"
+)
+
+var meta = persist.Metadata{
+	Version: "0.1",
```

Comment:
Put the version at the current release. It helps in the future when we're trying to figure out where things changed and when compatibility was updated or broken.

"0.4.0"